phpunit.xml.dist

```diff
@@ -7,10 +7,11 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
          bootstrap="tests/bootstrap.php">
 
     <testsuite name="Config Migration Tool Test Suite">
-        <directory>tests/Keboola</directory>
+        <directory>./tests/Keboola</directory>
+        <exclude>./tests/Keboola/ConfigMigrationTool/Migrations/KeboolaGoodDataWriterMigrationTest.php</exclude>
+        <exclude>./tests/Keboola/ConfigMigrationTool/Migrations/WrGoogleDriveMigrationTest.php</exclude>
```

Comment:
cim vic o tom premyslim, tim vic se me to nelibi :) markTestAsSkipped kdyz uz , tady v tom excludu na to 100% zapomenem a uz to tu navzdy zustane a ten test se spoustet nebude (ale stejne si myslim, ze by vubec nemelo byt v masteru)

Reply:
jj, vyhodim to